plugins/miscellanea/nanosound/install.sh

```diff
@@ -5,22 +5,21 @@ sudo apt-get update
 # Install the required packages via apt-get
 
 #START OF LIRC
-sudo apt-get -y install lirc
-echo "Applying LIRC starting policy"
-systemctl disable lirc.service
-systemctl stop lirc.service
-
-echo "Creating lircrc file"
-sudo touch /etc/lirc/lircrc
+#sudo apt-get -y install lirc
+#echo "Applying LIRC starting policy"
+#systemctl disable lirc.service
+#systemctl stop lirc.service
+sudo apt-get -y remove lirc
+
+#echo "Creating lircrc file"
+#touch /etc/lirc/lircrc
```

Comment:
so LIRC is completely out of the picture ? Does this mean that if LIRC needs to be installed for some other usage it's not compatible ?

Reply:
we have avoided the need to remove LIRC.  Nanosound does not depend on it, but it's not nessessary to remove it.